sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -75,56 +46,15 @@
   </SALCommand>
   <SALCommand>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_vea01P01_enable</EFDB_Topic>
-    <Description>Enable command for the vea01P01 device.</Description>
-    <item>
-      <EFDB_Name>comandoEncendido</EFDB_Name>
-      <Description>Command Started</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_vec01P01_enable</EFDB_Topic>
-    <Description>Enable command for the vec01P01 device.</Description>
-    <item>
-      <EFDB_Name>comandoEncendido</EFDB_Name>
-      <Description>Command Started</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_vin01P01_enable</EFDB_Topic>
-    <Description>Enable command for the vin01P01 device.</Description>
-    <item>
-      <EFDB_Name>comandoEncendido</EFDB_Name>
-      <Description>Command Started</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_crack01P02_enable</EFDB_Topic>
-    <Description>Enable command for the crack01P02 device.</Description>
+    <EFDB_Topic>HVAC_command_config_cracks</EFDB_Topic>
+    <Description>Configuration command for command for a single HVAC device in the CRACK device group.</Description>
```

Comment:
Please consider "Configure a single Crack device". (Or CRACK if it's an acronym). If you can provide a brief translation/description of what a Crack device is, that would also be helpful.

and similarly for the other configure commands.

Reply:
Even better, thanks! Done.